# P01 - QIIME 2 Importing, demultiplexing and clustering 
Feb 03rd, 2026

## Purpose: 
To import and demultiplex 16S rRNA sequences from microgravity dataset in the shared container in the server using QIIME2. To use the demux_seqs.qzv to determine if it is necessary to truncate/trim reads and by how much. Create files stasts.qza, table.qza, table.qzv, rep-seqs.qza and rep-seqs.qzv

## Material:
1. QIIME2 and shared server
2. microgravity_manifest.tsv (/datasets/project_2/microgravity/microgravity_manifest.tsv)
3. DADA2
4. demux_seqs.qzv
5. table.qza
6. rep-seqs.qza

## Procedure:
1. Connect to  Cisco Secure VPN. Open the terminal and login to MICB 475 class server using the provided shared container login credentials
2. Create dedicated directory for project 2: /data/microgravity_project
3. Import and demultiplex the dataset using paired-end code to create a qza file with demultiplexed sequences
4. Create demux_seqs.qzv for visualization and move to local computer to input in https://view.qiime2.org/ to decide how to trim/truncate reads
5. Determine ASVs and truncate/trim reads, and denoise and cluster the demultiplexed sequences using DADA2, creating table.qza, rep-seqs.qza, stats.qza
6. Create the visualization files table.qzv and rep-seqs.qzv
7. Convert qza files to qzv
8. Transfer the visualization files to local computer and view the representative sequences and table.qzv using https://view.qiime2.org/

## Code:
[QIIME2 Code pt.1](../Scripts/QIIME2_pt1.txt)

## Output files and path in the server: 
1.  /data/microgravity_project/demux_seqs.qza
2.  /data/microgravity_project/demux_seqs.qzv - [demux_seqs.qzv](../Backup_Files/demux_seqs.qzv)
3.  /data/microgravity_project/table.qza
4.  /data/microgravity_project/table.qzv - [table.qzv](../Backup_Files/table.qzv)
5.  /data/microgravity_project/rep-seqs.qza
6.  /data/microgravity_project/rep-seqs.qzv - [rep-seqs.qzv](../Backup_Files/rep-eqs.qzv)
7.  /data/microgravity_project/stats.qza

## Results:
### Before denoising (demux_seqs.qzv)
The same result is generated from samples sequenced from both forward and reverse primers
1. Total number of samples: 140
2. Range of sequencing depth: 61,351 - 191,114
3. Maximum read length (bp): 151 nts
4. 98% of samples have 151 nts

Reads quality score
<img src="/Images/Quality_score_fwd_reads.png">
<img src="/Images/Quality_score_rev_reads.png">

### After denoising (table.qzv)
The same result is generated from samples sequenced from both forward and reverse primers
1. Total number of samples: 140
2. Range of sequencing depth: 39,956 - 159,209
   
Attached images are generated from [https://view.qiime2.org/](https://view.qiime2.org/)

## Discussion:
1. Both forward and reverse reads were not trimmed at the beginning (left at 0nts), since median quality scores were all above 35. Forward reads were not truncated at the end, since median quality scores were all above 35 (left at 150nts). However, reverse reads were truncated at 140nts due to quality score drop.
2. Denoising did not remove any samples.


## Future directions:
The clustered ASVs will be filtered to exclude any mitochondrial or chloroplast sequences, which will remove any eukaryotic data.
